packages/cli/src/services/orchestration/main/handle-command-message-main.ts

```diff
@@ -53,11 +53,7 @@ export async function handleCommandMessageMain(messageString: string) {
 				}
 
 				if (isMainInstance && !config.getEnv('multiMainSetup.enabled')) {
-					// at this point in time, only a single main instance is supported, thus this command _should_ never be caught currently
-					logger.error(
```

Comment:
why delete the log? is this not likely to happen anymore?

Reply:
This was a stale log from pre-multi-main times. In single-main mode, this check prevents a main from reloading the license in response to a reload license command sent by that main itself.